### PR TITLE
CV UI Filter Create/Delete/Edit

### PR DESCRIPTION
**Pull request by sambible**
Will be in draft for a bit, need to make a few tweaks before I put this up for review. 

This will enable most actions surrounding filters, but only includes full entity support for one type of Filter ( RPM )
My plan here is to leave these locators in, but come back and exercise them as part of my Component Eval work, and move onto working on some of the few other areas remaining to enable that work.

### CODE REVIEW

**Comment by sambible** [comment]
@omkarkhatavkar @LadislavVasina1 @mstrlc Would appreciate some eyeballs here folks!

**Review by omkarkhatavkar — APPROVED** [review]
@sambible Can you please squash your commits, I see so many(14) commits.

File: airgun/entities/contentview_new.py [hunk 1]
```diff
@@ -57,6 +130,17 @@ def step(self, *args, **kwargs):
...
 
+@navigator.register(NewContentViewEntity, 'French')
+class ShowAllContentViewsScreenFrench(NavigateStep):
+    """Navigate to All Content Views screen ( in French )"""
+
+    VIEW = ContentViewTableView
+
+    @retry_navigation
+    def step(self, *args, **kwargs):
+        self.view.menu.select('Contenu', 'Lifecycle', 'Content Views')
```
Comment (adarshdubey-star) [inline airgun/entities/contentview_new.py:141]:
```suggestion
        self.view.menu.select('Content', 'Lifecycle', 'Content Views')
```

**Review by adarshdubey-star — APPROVED** [review]
LGTM

**Pull request merged**